Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Beyond the Dark Portal

## Changelog:
- Added the Beyond the Dark Portal event chain.
  - It fires for the Dark Portal holder of non-Draenor faith and culture.
  - [Full Design](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/issues/414) if you want to dive into it.
- Buffed Dark Portal defense.


# How to test:
@Warcraft-GoA-Development-Team/ck3-testers start as Stormwind, enable `yesman`, win the Horde, wait a bit of time for the event to trigger.

@Warcraft-GoA-Development-Team/english-proofreaders open the [Page](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/446/files?file-filters%5B%5D=.yml), scroll to the bottom to the `.yml` files, check them, suggest changes. If you forgot, here's the [Suggestion Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/How-to-be-Tester-(For-CK3)#reviewing-code).
